certbot-nginx/certbot_nginx/tests/nginxparser_test.py

```diff
@@ -281,18 +283,15 @@ def test_weird_blocks(self):
             [['return', '403']]], [['if', '($args', '~', 'post=140)'],
             [['rewrite', '^', 'http://example.com/']]],
             [['location', '~', '^/users/(.+\\.(?:gif|jpe?g|png))$'],
-            [['alias', '/data/w3/images/$1']]]]
+            [['alias', '/data/w3/images/$1']]],
+            ['proxy_set_header', 'X-Origin-URI', '${scheme}://${http_host}/$request_uri']]
```

Comment:
Lines 183, 229, 241, 280, and 345 all test that already, so I'd rather not add another line duplicating that check.

But more importantly, I'm a little worried that I didn't properly explain what changed here, because this PR doesn't change from only allowing `$var` to allowing `${var}`; it changes from only allowing `${var}` in the middle of a larger token to also allowing a token to start with `${`.

Reply:
OK, that sounds good then.